packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -12,7 +12,6 @@ import { URLInput } from '../';
 import LinkControlSearchResults from './search-results';
 import { CREATE_TYPE } from './constants';
 import useSearchHandler from './use-search-handler';
-
```

Comment:
Do you have some kind of formatter enabled on your code editor that is deleting these newlines? We should keep them in for readability.

Reply:
> Do you have some kind of formatter enabled on your code editor that is deleting these newlines? We should keep them in for readability.

Yeah, sorry about that, I tried to configure Prettier using the guidelines given, but could not do it. Can you share a resource to setup prettier and eslint properly?

Reply:
> Do you have some kind of formatter enabled on your code editor that is deleting these newlines? We should keep them in for readability.

Yes, I will fix this